src/ColorPalette/get-color.js

```diff
@@ -0,0 +1,30 @@
+const axe = require('axe-core');
+
+export default function getColor(hex) {
+	let i = 0
+	let chars = hex.toUpperCase().split('');
+
+  chars = chars.map((c) => {
```

Comment:
you should replace this with the rgbHex module I found

Reply:
Also, not really sure that it justifies its own module when you take this code away...